theme/src/components/teaser/_teaser_text_at_bottom.scss

```diff
@@ -118,4 +118,21 @@
       border-color: $color-shade-1;
     }
   }
+
+  //=================== Media queries ===================
+
+  @media (max-width: 767px) {
+    .cmp-teaser__action-link {
+      padding: 0.1em 0.7em;
+      width: 7em;
+      font-weight: bold;
+    }
+  }
+
+  @media (max-width: 767px) {
+    .cmp-teaser {
+      margin: 0 !important;
```

Comment:
Is it possible to avoid using `!important` ?

Reply:
Removed !important